src/rabbit_core_ff.erl

```diff
@@ -95,3 +103,19 @@ remove_explicit_default_bindings(FeatureName, Queues) ->
     [rabbit_binding:remove_default_exchange_binding_rows_of(Q)
      || Q <- Queues],
     ok.
+
+
+%% -------------------------------------------------------------------
+%% Virtual host metadata.
+%% -------------------------------------------------------------------
+
+virtual_host_metadata_migration(_FeatureName, _FeatureProps, enable) ->
+  Tab = rabbit_vhost,
+  rabbit_table:wait([Tab]),
```

Comment:
We must use the `rabbit_table:wait([Tab], _Retry = true)` (`Retry` defauls to `false`), otherwise the behavior is inconsistent during RabbitMQ startup: the general table wait already done by RabbitMQ retries. So if it takes time for the other nodes to come up, the migration will fail (no retry), but the general table wait will succeed (after retry), leading to inconsistent  state.